UIKItTemplate/Modules/TabBar/TabBarController.swift

```diff
@@ -0,0 +1,55 @@
+// TabBarController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// General app's tabBarController
+final class TabBarController: UITabBarController {
```

Comment:
```suggestion
final class MainTabBarController: UITabBarController {
```
General же сам написал

Reply:
Done